src/options.h

```diff
@@ -172,7 +174,8 @@ class options_manager : public JsonSerializer, public JsonDeserializer
         void add( const std::string sNameIn, const std::string sPageIn,
                   const std::string sMenuTextIn, const std::string sTooltipIn,
                   const int iMinIn, int iMaxIn, int iDefaultIn,
-                  copt_hide_t opt_hide = COPT_NO_HIDE );
+                  copt_hide_t opt_hide = COPT_NO_HIDE,
+                  const std::string &format = "%i" );
```

Comment:
Could not possibly agree more. It's a horrendously bad design and should probably be replaced en-masse. I'm looking into this for a separate PR.

For now though a minimal number of lines gets rid of a worse evil (spurious mods) and as a bonus provides two meaningful configuration options.